dlrover/python/common/constants.py

```diff
@@ -96,6 +96,37 @@ class NodeEventType(object):
     NODE_CHECK_FAILED = "NODE_CHECK_FAILED"
 
 
+class GpuMetricType(object):
+    """
+    it is the metrics of nvidia GPU, collected by DCGM
+    """
+
+    GPU_FREE_MEM = "DCGM_FI_DEV_FB_FREE"
+    GPU_USED_MEM = "DCGM_FI_DEV_FB_USED"
+    GPU_UTIL = "DCGM_FI_DEV_GPU_UTIL"
+    GPU_TEMP = "DCGM_FI_DEV_GPU_TEMP"
+    GPU_SM_UTIL = "DCGM_FI_PROF_SM_OCCUPANCY"
+    GPU_SM_ACTIVE = "DCGM_FI_PROF_SM_ACTIVE"
+    GPU_TENSOR_UTIL = "DCGM_FI_PROF_PIPE_TENSOR_ACTIVE"
+
+
+class NpuMetricType(object):
```

Comment:
There is overlap between NPU metrics and GPU metrics. Those metrics (e.g., FREE_MEM, UTIL, NUM) can be put into a separate class, e.g., ChipMetrics or something similar.